homeassistant/components/wolflink/sensor.py

```diff
@@ -79,6 +81,15 @@ def extra_state_attributes(self):
             "parent": self.wolf_object.parent,
         }
 
+    @property
+    def device_info(self) -> DeviceInfo:
```

Comment:
Last remark after my final review: You can use the shorthand attribute `_attr_device_info` in `__init__` instead of the property function.